samples/excel/worksheet/list-worksheets.yaml

```diff
@@ -0,0 +1,124 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: List worksheets
+description: Explore the API to list the worksheets in the workbook
+script:
+  content: |-
+    $("#list-worksheets").click(listWorksheets);
+
+    async function listWorksheets() {
+        try {
+            await Excel.run(async (context) => {
+                let sheets = context.workbook.worksheets;
+                sheets.load("items/name");
+                
+                await context.sync();
+
+                if (sheets.items.length > 1) {
+                    console.log(`There are ${sheets.items.length} worksheets in the workbook:`);
+                } else {
+                    console.log(`There is one worksheet in the workbook:`);
+                }
+                for (let i in sheets.items) {
+                    console.log(sheets.items[i].name);
```

Comment:
This is kinda boring.  Like, really boring.  Maybe make it add a button/hyperlink for each worksheet?  and when clicked , it does "sheet.activate()"